default.nix

```diff
@@ -20,6 +20,9 @@ mkDerivation {
     hnix io-streams optparse-applicative temporary text transformers
     unordered-containers
   ];
+  shellHook = ''
```

Comment:
can you show how to do it? I tried overriding from `shell.nix` and failed miserably.